libcomposefs/lcfs-writer.c

```diff
@@ -471,9 +471,16 @@ struct lcfs_node_s *lcfs_node_new(void)
 		errno = ENOMEM;
 		return NULL;
 	}
-
+	
+	memset( node, 0, sizeof(struct lcfs_node_s));
+	node->payload = NULL;
+	node->content = NULL;
 	node->ref_count = 1;
 	node->inode.st_nlink = 1;
+	node->delayedDigestCalculation = false;
+	node->delayedInline = false;
+	node->delayedPayload = false;
+	node->delayedResetDigest = false;
```

Comment:
Additionally, we generally rely on the zero initializing to do all these. Only non-zero initializers are needed.

Reply:
This is not required anymore.